script/generate_needle_previews

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/perl -w
+# Copyright 2025 Roland Clobus <rclobus@rclobus.nl>
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+
+=head1 generate_needle_previews
+
+generate_needle_previews - generate .svg files with the needle information as an overlay
+
+=head1 SYNOPSIS
+
+cd products/XXX/needles
+ls -1 "*.json" | generate_needle_previews
+
+=cut
+
+use v5.10;
+use strict;
+use warnings;
+use autodie;
+use Image::PNG;
```

Comment:
So we don't have Image::PNG in our dependencies.
How should we handle that?

Reply:
We could add an optional dependency. It isn't in TW so I wouldn't make it a hard dependency.